src/applications/check-in/pre-check-in/tests/e2e/happy-path/happy.path.cypress.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Pre-Check In Experience ', () => {
     // page: Confirmation
     cy.get('h1', { timeout: Timeouts.slow })
       .should('be.visible')
-      .and('have.text', 'Confirmation');
+      .and('have.text', 'You’ve completed pre check-in');
```

Comment:
You should use the Confirmation page helper here.

Reply:
Updated in commit [1233dad](https://github.com/department-of-veterans-affairs/vets-website/pull/19570/commits/1233dad1e575ab03a669892a49d307c2801b2dbc)